__test__/topic-graph-test.js

```diff
@@ -96,4 +96,38 @@ describe('topic graph', () => {
       ]);
     });
   });
+
+  describe('with a pub/sub and local pub/sub', () => {
+    const result = graph(simpleLocal.services, simpleLocal.blocks);
+
+    it('should have four nodes', () => {
+      expect(result.nodes).to.have.length(4);
+      expect(result.nodes).to.have.members([
+        'PubService', 'SubService', 'LocalPubService', 'LocalSubService'
+      ]);
+    });
+
+    it('should have two edges', () => {
+      expect(result.edges).to.deep.equal([
+        ['PubService', 'SubService', 'nio.data'],
+        ['LocalPubService', 'LocalSubService', 'nio.data']
+      ]);
+    });
+
+    it('should find localPublishers through `nio.data`', () => {
+      const localPubs = result.localPublishersOf('nio.data');
```

Comment:
Should maybe add a `result.publishersOf('nio.data')` test as well to make sure the local ones don't collide with that.